templates/autoscale-tgw-new-vpc.template.yaml

```diff
@@ -7,362 +7,119 @@ Description: >-
-            uppercase letters, period (.), and hyphens (-). It cannot start or
-            end with a hyphen (-).
-    S3KeyPrefix:
+    AutoscaleNotificationSubscriberEmail:
+        AllowedPattern: ^$|^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$
+        ConstraintDescription: must be a valid email address.
+        Description: The email address (AWS SNS Topic subscriber) to receive Autoscale notifications. If provided, the template can only accept one email address. An email will be sent to the address to confirm the subscription.
         Type: String
-        AllowedPattern: '^$|^[0-9a-zA-Z-./]*/$'
-        ConstraintDescription: >-
-            Deployment package S3 key prefix can include numbers, lowercase
-            letters, uppercase letters, hyphens (-), period (.), and forward
-            slash (/). If provided, it must end with a forward slash (/).
-        Description: >-
-            Name of the S3 folder that stores the FortiGate Autoscale deployment
-            resources. Can only contain numbers, lowercase letters, uppercase
-            letters, hyphens (-), period (.), and forward slashes (/). If
-            provided, it must end with a forward slash (/).
     AvailabilityZones:
-        Description: >-
-            The list of Availability Zones to use for the subnets in the VPC.
-            The FortiGate Autoscale solution uses two Availability Zones from
-            your list and preserves the logical order you specify.
-        Type: 'List<AWS::EC2::AvailabilityZone::Name>'
-    VpcCidr:
-        Type: String
-        Default: 192.168.0.0/16
-        AllowedPattern: >-
-            ^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]){1}(\/([0-9]|[1-2][0-9]|3[0-2]))?$
-        ConstraintDescription: must be a valid CIDR block format.
-        Description: The CIDR block for the FortiGate Autoscale VPC.
-    PublicSubnet1Cidr:
-        Type: String
-        Default: 192.168.0.0/24
-        AllowedPattern: >-
-            ^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]){1}(\/([0-9]|[1-2][0-9]|3[0-2]))?$
-        ConstraintDescription: must be a valid CIDR block format.
-        Description: >-
-            The CIDR block for the subnet located in Availability Zone 1 where
-            the FortiGate Autoscale instances will be deployed to.
-    PublicSubnet2Cidr:
+        Description: The list of Availability Zones to use for the subnets in the VPC. The FortiGate Autoscale solution uses two Availability Zones from your list and preserves the logical order you specify.
+        Type: List<AWS::EC2::AvailabilityZone::Name>
+    BgpAsn:
+        Default: 65000
+        Description: The Border Gateway Protocol (BGP) Autonomous System Number (ASN) of the Customer Gateway of each FortiGate-VM instance in the Auto Scaling group. This value ranges from 64512 to 65534.
```

Comment:
```suggestion
        Description: The Border Gateway Protocol (BGP) Autonomous System Number (ASN) of the customer gateway of each FortiGate-VM instance in the Auto Scaling group. This value ranges from 64512 to 65534.
```

Reply:
This is treated as a name. no need to change.